modules/account-lib/src/coin/cspr/keyPair.ts

```diff
@@ -43,26 +43,17 @@ export class KeyPair extends Secp256k1ExtendedKeyPair {
       return {
         pub: HDNode.fromBase58(xpub)
           .getPublicKeyBuffer()
-          .toString('hex')
-          .toUpperCase(),
+          .toString('hex'),
         prv: xprv

```

Comment:
Should "ToLowerCase" be used instead of removing "ToUpperCase"?
The same for the other cases.